.github/workflows/markdown-lint.yml

```diff
@@ -0,0 +1,15 @@
+name: markdown-lint
+
+on: [push, pull_request]
```

Comment:
Should we add [`paths`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths) to run this workflow only when `README.md` has been modified?

Reply:
done